zebra-chain/src/sapling/arbitrary.rs

```diff
@@ -99,3 +102,43 @@ impl Arbitrary for OutputInTransactionV4 {
 
     type Strategy = BoxedStrategy<Self>;
 }
+
+/// A wrapper for a RedJubJub spending verification key.
+///
+/// This is used by proptests since we can't implement Arbitrary for the
+/// redjubjub::VerificationKeyBytes (it's in another crate).
+/// We then implement it for the wrapper type instead.
+#[derive(Debug)]
+struct SpendVerificationKeyBytesWrapper(redjubjub::VerificationKeyBytes<redjubjub::SpendAuth>);
+
+impl Arbitrary for SpendVerificationKeyBytesWrapper {
+    type Parameters = ();
+
+    fn arbitrary_with(_args: Self::Parameters) -> Self::Strategy {
+        (prop::array::uniform32(any::<u8>()))
+            .prop_map(|bytes| {
+                let mut rng = ChaChaRng::from_seed(bytes);
```

Comment:
Comment: it's unfortunate that `VerificationKey::from(&jubjub::Scalar)` isn't available outside of the crate :( That would have allowed using the same `from_bytes_wide` trick 😁 

(https://docs.rs/redjubjub/0.4.0/src/redjubjub/verification_key.rs.html#139-146)

(Note: feel free to resolve this remark 👍)

Reply:
Yeah :cry: